tests/libkubevirt/config/kvconfig.go

```diff
@@ -79,6 +79,7 @@ func patchKV(namespace, name string, patchSet *patch.PatchSet) error {
 func UpdateKubeVirtConfigValueAndWait(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 	kv := testsuite.UpdateKubeVirtConfigValue(kvConfig)
 
+	testsuite.EnsureKubevirtReady()
```

Comment:
I believe this was the missing bit;
It ensures that the `/healthz` requests can start going through, since the kubevirt operator has both processed the desired state change (generation vs observedGeneration) and the `Available` condition is `True`.